WordPress/Classes/Services/MediaService.m

```diff
@@ -4,15 +4,18 @@
 #import "WPAccount.h"
 #import "ContextManager.h"
 #import "Blog.h"
-#import "UIImage+Resize.h"
 #import <MobileCoreServices/MobileCoreServices.h>
 #import "WordPress-Swift.h"
 #import "WPXMLRPCDecoder.h"
 #import <WordPressShared/WPImageSource.h>
 #import <WordPressShared/WPAnalytics.h>
+
 @import WordPressKit;
+@import WordPressUI;
 @import WordPressShared;
 
+
```

Comment:
Do we have it specified somewhere that we want three empty lines between imports and the beginning of our code?

Reply:
Good question. No, i don't think so. Should i revert this one?

Reply:
In this PR extra lines were added to several files, not really a big deal, but we should make up our minds and document it somewhere.

Reply:
Sounds like a job for Guidelines! will try to send a PR next week!